icicle/src/fields/mont.cu

```diff
@@ -0,0 +1,19 @@
+#include "fields/field_config.cuh"
+
+using namespace field_config;
+
+#include "utils/mont.cuh"
+#include "gpu-utils/device_context.cuh"
+#include "utils/utils.h"
+
+namespace mont {
+  extern "C" cudaError_t CONCAT_EXPAND(FIELD, ScalarConvertMontgomery)(
```

Comment:
I wonder if we should expose all the C-API declarations in header files. It's of course relevant to all exposed APIs. I realize that ffi is declaring it too but to make the exposed APIs clear maybe it's worth doing so.
What do you think?

Reply:
Yes, I think we can try it

Reply:
Maybe we could add a include/c_api dir and place all those headers there. It would be quite clear what is exposed without looking for it in src dirs. I think better have different files, for NTT, MSM etc. rather than one for all